com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs

```diff
@@ -65,7 +65,7 @@ protected override bool DoShadersStripper(HDRenderPipelineAsset hdrpAsset, Shade
             // If requested by the render pipeline settings, or if we are in a release build,
             // don't compile fullscreen debug display variant
             bool isFullScreenDebugPass = snippet.passName == "FullScreenDebug";
-            if (isFullScreenDebugPass && (!Debug.isDebugBuild || !hdrpAsset.currentPlatformRenderPipelineSettings.supportRuntimeDebugDisplay))
+            if (isFullScreenDebugPass && (!Debug.isDebugBuild || !HDRenderPipelineGlobalSettings.instance.supportRuntimeDebugDisplay))
```

Comment:
you should probably use `globalSettings` instead of `HDRenderPipelineGlobalSettings.instance`